src/app/cases/styles.tsx

```diff
@@ -16,7 +16,7 @@ export const FiltersContainer = styled.div`
 
```

Comment:
I think we should also shift from the current box shadow on the big card to the bordered design!

Current | Figma
:--------:|:-------:
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/9daaf81b-2e30-4ac9-80f9-57e0ec7b6352) | ![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/04ea25c0-9143-4e4a-9992-41d213c9f6ff)

Reply:
this should be fixed in my other PR #25!